docs/apache-airflow-providers-amazon/operators/ecs.rst

```diff
@@ -46,6 +46,149 @@ Before using EcsOperator *cluster*, *task definition*, and *container* need to b
     :start-after: [START howto_operator_ecs]
     :end-before: [END howto_operator_ecs]
 
+Using Operator
+--------------
+
+
+Launch Types
+--------------
+
+You can use this Operator to run ECS Tasks in ECS Clusters with launch types of EC2, Fargate and EXTERNAL, using the "launch_type" parameter. Bear in mind that the different launch types will require different parameters to be supplied.
+
+```
+launch_type="EC2|Fargate|EXTERNAL"
+```
+
+*Example Operator for launch_type of EC2 and EXTERNAL*
+
+```
+    hello_world = EcsOperator(
+        task_id="hello_world",
+        dag=dag,
+        aws_conn_id="aws_default",
+        cluster="ecs-cluster-name",
+        task_definition="ecs-task-definition",
+        launch_type="EC2|EXTERNAL",
+        overrides={ "containerOverrides": [
+            {
+                "name": "hello-world-container",
+                "command" : [ "echo","hello world from Airflow" ],
+            }
+        ] },
+        tags={
+            "Customer": "X",
+            "Project": "Y",
+            "Application": "Z",
+            "Version": "0.0.1",
+            "Environment": "Development",
+            }
+    )
```

Comment:
Code examples are preferably to come from the example dags.
There is no need to hard code them in the doc. Check other operators examples

Reply:
Its ok, I am just looking at some of the other Amazon providers (EMR) and it looks like they do something like what I need so I will copy their approach. Thanks.

Reply:
proceed with how you see fit and we can check it during review.

Reply:
All done, checked in with no errors this time.